argo/workflow/tiflash.yaml

```diff
@@ -0,0 +1,59 @@
+metadata:
+  generateName: tipocket-tiflash-
+  namespace: argo
+spec:
+  entrypoint: call-tipocket-tiflash
+  onExit: call-export-logs
+  arguments:
+    parameters:
+      - name: ns
+        value: tipocket-tiflash
+      - name: purge
+        value: "false"
+      - name: image_version
+        value: release-4.0-nightly
+      - name: storage_class
+        value: pd-ssd
+      - name: nemesis
+        value: ""
+      - name: run_time
+        value: "6h"
+      - name: tiflash_image
+        value: "yeya24/tics:4.0"
```

Comment:
Can we use official image

Reply:
No. The official has some problems in its image. So I have to build my own. I think it is okay to update it later